cat-launcher/src/hooks/useDeleteBackup.ts

```diff
@@ -0,0 +1,16 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+import { deleteBackupById } from "@/lib/commands";
+import { queryKeys } from "@/lib/queryKeys";
+import { GameVariant } from "@/generated-types/GameVariant";
+
+export function useDeleteBackup(variant: GameVariant | null) {
+  const queryClient = useQueryClient();
+
+  return useMutation({
```

Comment:
Perform an optimistic update. If the actual update fails, display an error using toastCL function, then revert the optimistic update.